### DISCUSSION TITLE
Ch08 - TodoItem non ispezionabile dal DevTool di React

### DISCUSSION THREAD

**Discussion by alessandropecchini99** [lifeisfoo/react-guida-pratica#20] [Q&A]
Forse è una domanda inutile, ma quando provo a ispezionare un TodoItem dal DevTool di React non mostra quello che contiene
Mi esce questo errore:

![Screenshot 2024-03-17 192344](https://github.com/lifeisfoo/react-guida-pratica/assets/85944525/7ba7d81f-98db-46bc-8958-f4f698c99548)

Come mai? C'è qualcosa che non va o è normale?

**Comment by lifeisfoo** [comment]
Ciao @alessandropecchini99,
- puoi condividere i tuoi package.json e package-lock.json?
- che versione di React stai usando?
- che strumento stai usando per eseguire l'applicazione? CRA? Vite? altro?

**Comment by alessandropecchini99** [comment]
Certo! Scusa la poca chiarezza
Sto usando react 18.2.0 dalla cartella ch07 del branch di Vite, ed ho avviato il server con il comando "npm start"
Nel mentre sono andato comunque avanti e nella cartella ch08 il problema non si presenta

Di seguito lascio i package:
[package.json](https://github.com/lifeisfoo/react-guida-pratica/files/14628896/package.json)
[package-lock.json](https://github.com/lifeisfoo/react-guida-pratica/files/14628897/package-lock.json)

**Reply by lifeisfoo** [reply]
Ho provato ad eseguire tutti e due i progetti (ch07 e ch08) nel branch Vite, usando Chrome, ma con nessuno dei due si verifica il problema.

Sinceramente me l'aspettavo perché credo che l'errore sia legato ai dev tools ed è probabile che ci sia qualcosa che non funziona bene tra questi e il browser. Facendo una veloce ricerca online ho trovato [lo stesso problema su StackOverflow](https://stackoverflow.com/q/70897741/3340702) ma le soluzione sono le più disparate. Farei comunque un tentativo.

Se ci fosse un errore sulla parte di React l'applicazione non funzionerebbe e se anche ci fosse qualcosa di non perfetto nell'app i dev tools dovrebbero permetteri di vederlo (non servono forse a questo?).

Fammi sapere se riesci a risolvere il problema e mi dispiace non poter essere d'aiuto più di cose.

**Reply by alessandropecchini99** [reply]
Grazie mille per la risposta e nessun problema, anch'io vedevo diverse soluzioni e non mi era chiara la situazione, per questo ho chiesto
Riprovando ora funziona... per cui dispiace a me di aver fatto perdere tempo e ti ringrazio ancora.
